src/Exporter/Element/ExpElement.php

```diff
@@ -76,6 +85,28 @@ public function getSerialization() {
 	}
 
 	/**
+	 * @see ExpElement::newFromSerialization
+	 */
+	protected static function deserialize( $serialization ) {
```

Comment:
Not a fan of having such static deserialization functions, same goes for the `ewFromSerialization` below